src/stm.js

```diff
@@ -22,6 +22,9 @@ var RECORDING_TIMEOUT = 3000;
 var RECORDING_BITS_PER_SECOND = 16000;
 
 var RECORDING_MIME_TYPE = 'audio/ogg';
+if (!MediaRecorder.isTypeSupported(RECORDING_MIME_TYPE)){
+  RECORDING_MIME_TYPE = 'audio/webm;codecs=opus';
+}
```

Comment:
Should we just default to webm, as it's more widely supported? What are the reasons for picking one over the other?

Reply:
Firefox doesn't support webm, only ogg, whilst Chrome supports webm but not ogg. The goal for this patch is to start supporting both FF and Chrome with the same codebase.